### PR TITLE
Save byte array allocations

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 @inject IContentManager ContentManager
 
 @{
-    var matchAllQuery = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
+  var matchAllQuery = OrchardCore.Queries.Sql.ViewModels.AdminQueryViewModel.MatchAllQueryBase64;
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml:6]:
```suggestion
  var matchAllQuery = AdminQueryViewModel.MatchAllQueryBase64;
```

File: src/OrchardCore.Modules/OrchardCore.Queries/Sql/ViewModels/AdminQueryViewModel.cs [hunk 2]
```diff
@@ -6,6 +6,8 @@ namespace OrchardCore.Queries.Sql.ViewModels
 {
     public class AdminQueryViewModel
     {
+        public static string MatchAllQueryBase64 { get; } = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Queries/Sql/ViewModels/AdminQueryViewModel.cs:9]:
```suggestion
        public static string MatchAllQueryBase64 => Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
```

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Queries/Sql/ViewModels/AdminQueryViewModel.cs (hunk 2 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Queries/Sql/ViewModels/AdminQueryViewModel.cs:9]: That would evaluate on each call. Which was done before this change.

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml:6]: It's not part of the usings

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml:6]: Sorry, I mean to use the model, I updated the above suggestion

**Review by MikeAlhayek — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml:6]: Static properties can't be accessed from instance references.

**Pull request merged**